hardware/mcp3008/pimcp3008.js

```diff
@@ -28,23 +28,26 @@ module.exports = function(RED) {
         this.dnum = parseInt(n.dnum || 0);
         this.bus = parseInt(n.bus || 0);
         this.dev = n.dev || "3008";
+	this.fs = require('fs');
```

Comment:
indent ->

Reply:
Here I was not shure if the definition from the global place caused any problems. So I createt a new one and renamed the orginal to fs2.
If you want to remove the line you have also edit line 42.
I testet this on my raspberry and it worked
from:
`
this.fs.statSync("/dev/spidev"+node.bus+"."+node.dnum);
`
to:
`
fs.statSync("/dev/spidev"+node.bus+"."+node.dnum);
`
I testet this on my raspberry and it worked